src/pages/docs/oauth/advanced/connection-context.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: OAuth Connection Context
+description: This describes the connection context
+tags:
+  - ShipEngine Connect
+  - OAuth
+---
+
+# Connection Context
+
+Nonstandard OAuth integrations occasionally both send and expect more than an
+access token in the response and authenticated requests, respectively.
+
+To accommodate for this, MonoAuth allows a response transformer to specify
+a collection of properties in `connection_context`. Properties defined in
+`connection_context` are included in the 'code for token' (`POST /authflow/token`)
+exchange response (the
```

Comment:
give an example of a monoauth response here, as the link to the C# POCO won't work :)